### DISCUSSION TITLE
Lenstube x Push

### DISCUSSION THREAD

**Discussion by sskidgoku** [tapexyz/tape#570] [Ideas]
Hello LensTube Community ,

### **Summary**

We propose the **Push Protocol** be integrated with LensTube.

Decentralized notifications through Push establish a direct line of communication between LensTube and its investors.

### **Background**

**[Push 2](https://push.org/)** is the first decentralized communication protocol that enables any dApp, smart contract, or backend to send push notifications tied to user wallet addresses.

Currently, the Push protocol powers notifications for 70+ projects, such as **Ethereum Name Service (ENS), Decentraland, Uniswap, Pine Protocol, and DoinGud** around the Ethereum ecosystem and soon will be deployed to all the EVM-compatible networks.

### **A brief history of Push Protocol**

We started building Push in January 2020.

While interacting with decentralized applications, we realized the need for a notification mechanism to notify users or wallet addresses about important events, updates, and actions. As a result, many essential events requiring user attention should be included.

Push tries to solve this with its **Push Notifications** service. To do this efficiently, the notification is stored and treated like a JSON payload, which is transformed as per the rules of the different carriers when the notification reaches them.

The JSON payload can differ with payload types, ensuring the content's flexibility and the data, storage interpretation, and delivery.

Push lets users control entirely what services they get notifications from and impose rules on the services, including spam protection for users. This means that notifications are treated as a social feed rather than a piece of information.

### **Why integrate Push with LensTube**

Making the best use of technologies at hand to enhance user experience is the essence of innovation in Web3. Many protocols and dApps currently rely on emails or social media tools like Twitter, Telegram, and Discord to communicate with their users.

Implementing Push’s decentralized notifications creates a direct channel between LensTube and **Users**. Users will be able to receive notifications and respond immediately.

All notifications can be delivered in rich media format, resulting in a more engaging user experience. Here are the exact use cases for which users of the Lenstube platform will receive notifications;

- Updates to users
- New products released
- Push Chat features

Sending out notifications for these events will be very beneficial for users in staying up-to-date with their current positions and taking appropriate actions.

### **Next steps**

Please feel free to share feedback with us and let us know your thoughts.

### **Link to relevant resources**

1. [[Push Docs](https://docs.push.org/hub/)](https://docs.push.org/hub/)

**Comment by atahanonkal** [comment]
its good
